share/doc/homebrew/Analytics.md

```diff
@@ -0,0 +1,41 @@
+# Analytics
+Homebrew will shortly begin gathering anonymous aggregate user behaviour analytics and reporting these to Google Analytics.
+
+## Why?
+Homebrew is provided free of charge and run entirely by volunteers in their spare time. As a result, we do not have the resources to do detailed user studies of Homebrew users to decide on how best to design future features and prioritise current work. Anonymous aggregate user analytics allow us to prioritise fixes and features based on how, where and when people use Homebrew.
+
+## What?
+Homebrew's analytics record some shared information for every event:
+
+- The Homebrew user agent e.g. `Homebrew/0.9.9 (Macintosh; Intel Mac OS X 10.11.4) curl/7.43.0`
+- The Google Analytics version i.e. `1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#v)
+- The Homebrew analytics tracking ID e.g. `UA-75654628-1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#tid)
+- A Homebrew analytics user ID e.g. `1BAB65CC-FE7F-4D8C-AB45-B7DB5A6BA9CB`. This is generated by `uuidgen` and stored in `~/.homebrew_analytics_user_uuid`. This does not allow us to track individual users but does enable us to accurately measure user counts vs. event counts (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#cid)
+- The Google Analytics anonymous IP setting is enabled i.e. `1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#aip)
+- The Homebrew application name e.g. `Homebrew` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#an)
+- The Homebrew application version e.g. `0.9.9` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#av)
+- The Homebrew analytics hit type e.g. `screenview` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#t)
+
+Homebrew's analytics records the following different events:
+
+- a `screenview` hit type with the Homebrew command you have run (with arguments stripped) e.g. `brew list` (not `brew list foo`)
+- an `event` hit type with the `install` event category, the Homebrew formula you have requested to install e.g. `ack` as the action and an event label e.g. `Mac OS X 10.11, non-/usr/local, CI=1`
+- an `event` hit type with the `BuildError` event category, the Homebrew formula that failed to install e.g. `wget` as the action and an event label e.g. `Mac OS X 10.11, non-/usr/local, CI=1`
```

Comment:
Here and one line above: `CI` instead of `CI=1`.

Also, maybe worth explaining how the label `Mac OS X 10.11, non-/usr/local, CI` is constructed? (Or maybe a bit further up in the “What?” section additional points like “non-standard installation location” and “whether or not invoked as part of CI”?)